apps/web/components/eventtype/EventTypeDescriptionSafeHTML.tsx

```diff
@@ -1,11 +1,14 @@
+import { addListFormatting } from "@calcom/lib/markdownIt";
+
 export type EventTypeDescriptionSafeProps = {
   eventType: { description: string | null };
 };
 
 export const EventTypeDescriptionSafeHTML = ({ eventType }: EventTypeDescriptionSafeProps) => {
   const props: JSX.IntrinsicElements["div"] = { suppressHydrationWarning: true };
-  // @ts-expect-error: @see packages/prisma/middleware/eventTypeDescriptionParseAndSanitize.ts
-  if (eventType.description) props.dangerouslySetInnerHTML = { __html: eventType.descriptionAsSafeHTML };
+  if (eventType.description)
+    // @ts-expect-error: @see packages/prisma/middleware/eventTypeDescriptionParseAndSanitize.ts
+    props.dangerouslySetInnerHTML = { __html: addListFormatting(eventType.descriptionAsSafeHTML) };
```

Comment:
I think addListFormatting should be part of markdownAndSanitize only as it's working with the generated HTML.


I don't know why bio doesn't support lists, but if it does it would also automatically show lists with right styles